File: ReSEP3-T1-T2/Model/Exercise.cs
using System.Text.Json.Serialization;

namespace Model
{
    public class Exercise
    {
        [JsonPropertyName("eId")] public int EId { get; set; }
        [JsonPropertyName("EName")] public string EName { get; set; }
        [JsonPropertyName("nrOfSets")] public int NrOfSets { get; set; }
        [JsonPropertyName("nrOfReps")] public int NrOfReps { get; set; }

        [JsonPropertyName("WorkoutPlanID")] public int WorkoutPlanId { get; set; }


        public Exercise()
        {
        }

        public Exercise(string eName, int nrOfSets, int nrOfReps, int workoutPlanId)
        {
            EName = eName;
            NrOfSets = nrOfSets;
            NrOfReps = nrOfReps;
            WorkoutPlanId = workoutPlanId;
        }
 
        public Exercise(int eid,string eName, int nrOfSets, int nrOfReps, int workoutPlanId)
        {
            EName = eName;
            NrOfSets = nrOfSets;
            NrOfReps = nrOfReps;
            WorkoutPlanId = workoutPlanId;
            EId = eid;
        }

        public Exercise(int eId, string eName, int nrOfSets, int nrOfReps)
        {
            EId = eId;
            EName = eName;
            NrOfReps = nrOfReps;
            NrOfSets = nrOfSets;
        }
    }
}

File: ReSEP3-T3/src/main/java/resep3/javat3/model/Exercise.java
package resep3.javat3.model;

public class Exercise {

    private int Eid;
    private String EName;
    private int NrOfSets;
    private int NrOfReps;
    private int WorkoutId;

    public Exercise(String EName, int nrOfSets, int nrOfReps, int workoutId) {

        this.EName = EName;
        NrOfSets = nrOfSets;
        NrOfReps = nrOfReps;
        WorkoutId = workoutId;
    }

    public Exercise() {

    }

    public int getEid() {
        return Eid;
    }

    public void setEid(int eid) {
        Eid = eid;
    }

    public String getEName() {
        return EName;
    }

    public void setEName(String EName) {
        this.EName = EName;
    }

    public int getNrOfSets() {
        return NrOfSets;
    }

    public void setNrOfSets(int nrOfSets) {
        NrOfSets = nrOfSets;
    }

    public int getNrOfReps() {
        return NrOfReps;
    }

    public void setNrOfReps(int nrOfReps) {
        NrOfReps = nrOfReps;
    }

    public int getWorkoutId() {
        return WorkoutId;
    }

    public void setWorkoutId(int workoutId) {
        WorkoutId = workoutId;
    }
}
